src/components/projects-summary/projects-summary.html

```diff
@@ -1,11 +1,75 @@
 <div class="catalogs-projects-summary-panel">
-  <button class="create-button btn btn-primary">
+  <button ng-if="$ctrl.canCreate" class="create-button btn btn-primary" ng-click="$ctrl.openNewProjectPanel()">
     <span class="fa fa-plus"></span>
     <span class="create-button-text">Create Project</span>
   </button>
-  <h2>Getting Started</h2>
-  <div class="getting-started-panel">
-    {{$ctrl.gettingStartedText}}
+  <div class="catalogs-create-project-panel" ng-if="$ctrl.showNewProjectPanel">
+    <div class="arrow"></div>
+    <div class="catalogs-create-project-title">
+      <span>Create Project</span>
+      <a class="pull-right" ng-click="$ctrl.closeNewProjectPanel()">
+        <span class="catalogs-create-project-close pficon pficon-close"></span>
+      </a>
+    </div>
+    <create-project alerts="$ctrl.alerts" is-dialog="true" redirect-action="$ctrl.onNewProject" on-cancel="$ctrl.closeNewProjectPanel"></create-project>
+  </div>
+
+  <div ng-if="$ctrl.showGetStarted">
+    <h2>Getting Started</h2>
+    <div class="getting-started-panel">
+      {{$ctrl.gettingStartedText}}
+    </div>
+  </div>
+  <div ng-if="!$ctrl.showGetStarted" class="catalogs-project-summary-list">
+    <div ng-if="$ctrl.maxDisplayProjectsReached" class="projects-count-area">
+      <span class="counter-text">{{$ctrl.maxDisplayProjects}}</span>
+      <span>of</span>
+      <span class="counter-text">{{$ctrl.totalProjects}}</span>
+      <span>Projects</span>
+      <a href="#" class="projects-view-all" ng-click="$ctrl.showAllProjects()">View All</a>
+    </div>
+    <div id="catalogs-projects-summary-list">
+      <div class="catalogs-edit-project-panel" ng-if="$ctrl.showEditProjectPanel" style="top: {{$ctrl.editPanelTopPosition}}px;">
+        <div class="arrow"></div>
+        <div class="catalogs-create-project-title">
+          <span>Edit Project - </span>
+          <span>{{$ctrl.edittingProject | displayName}}</span>
+          <a class="pull-right" ng-click="$ctrl.closeEditProjectPanel()">
+            <span class="catalogs-create-project-close pficon pficon-close"></span>
+          </a>
+        </div>
+        <edit-project project="$ctrl.edittingProject" is-dialog="true" alerts="$ctrl.alerts" redirect-action="$ctrl.onEditProject" on-cancel="$ctrl.closeEditProjectPanel"></edit-project>
+      </div>
+      <div ng-repeat="project in $ctrl.projects track by $index" class="project-tile">
+        <div class="dropdown  dropdown-kebab-pf" uib-dropdown="">
+          <button class="btn btn-link uib-dropdown-toggle" type="button" id="dropdownKebab" aria-haspopup="true" aria-expanded="true" uib-dropdown-toggle>
+            <span class="fa fa-ellipsis-v"></span>
+          </button>
+          <ul class="uib-dropdown-menu dropdown-menu-right" aria-labelledby="dropdownKebab">
+            <li><a href="#" ng-click="$ctrl.editProject(project, 'catalogs-summary-project-' + $index)">Edit</a></li>
```

Comment:
Are we going to have problems where the dialog is in the wrong place if the list changes during an edit?